netmaster/main.go

```diff
@@ -177,13 +177,29 @@ func (d *daemon) execOpts() {
 }
 
 func (d *daemon) ListenAndServe() {
-	r := mux.NewRouter()
+	router := mux.NewRouter()
 
 	// Create a new api controller
-	d.apiController = objApi.NewAPIController(r)
+	d.apiController = objApi.NewAPIController(router)
+
+	// Setup the router to serve the web UI
+	goPath := os.Getenv("GOPATH")
```

Comment:
GOPATH might not always be a good way to get the source directory, especially in deployment based on just binary installation. Also it might break if we were to test this in systemtests that use godep, which changes the GOPATH.

May be we can introduce a command-line flag or a configuration value to netmaster that takes the site path?